docs/rest_api_2.rst

```diff
@@ -0,0 +1,4156 @@
+````````````````````
+.. code-block:: json
+   
+   Key           Value           
+   Content-Type: application/json
+   
+Input Parameters
+----------------
+
+There are multiple F5 ACI ServiceCenter APIs which require input parameters to be retrieved for the APIC Logical Device. These input parameters can be retrieved from APIC object browser known as visore.html
+
+APIC managed objects can be accessed at https://<APIC_IP>/visore.html.
+
+Below is a table of such parameters and steps on how to retrieve them.
+These parameters will be required for L2-L3 Stitching tab.
+
++-------------+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Input Key   | Input Value                                                                                                                    | Input Value Example                                                                                                                                     |
++=============+================================================================================================================================+=========================================================================================================================================================+
+| ldev        | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device>                                                      |     uni/tn-Sample\_2/lDevVip-f5-gs                                                                                                                      |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the <Your L4-L7 Device> and click on “Open in Object Store Browser”                                          |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Use the dn property from the object browser for input parameter “ldev”.                                                     |                                                                                                                                                         |
```

Comment:
the input parameter

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)